src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract: function (num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply: function (num) {
+      this.result *= num;
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - Use the method shorthand syntax for defining methods in an object for consistency and cleaner code.

Replace:
multiply: function(num) {
  // your code
},

With:
multiply(num) {
  // your code
},